sentry/src/transports/tokio_thread.rs

```diff
@@ -7,6 +7,7 @@ use std::time::Duration;
 use super::ratelimit::{RateLimiter, RateLimitingCategory};
 use crate::{sentry_debug, Envelope};
 
+#[expect(clippy::large_enum_variant)]
 enum Task {
     SendEnvelope(Envelope),
```

Comment:
Also here, let's try to remove the `large_enum_variant` warning if possible.

```suggestion
enum Task {
    SendEnvelope(Box<Envelope>),
```